bin/correctness/ground-truth/src/config.rs

```diff
@@ -20,12 +20,21 @@ pub struct Cli {
     #[arg(long, default_value = "/usr/local/bin/millstone")]
     pub millstone_binary_path: String,
 
-    /// Path to the millstone configuration file to use.
+    /// Path to the millstone configuration file to use for the non ADP group.
     ///
-    /// This file is mapped into the millstone container and so it must exist on the system where this command is run
-    /// from.
+    /// This file is mapped into the DSD group's millstone container and so it must exist on the system where this
+    /// command is run from.
+    #[arg(long)]
+    pub dsd_millstone_config_path: PathBuf,
+
+    /// Optional path to the millstone configuration file to use for the ADP group.
+    ///
+    /// If not specified, uses the same config as the non ADP group.
+    ///
+    /// This file is mapped into the ADP group's millstone container and so it must exist on the system where this
+    /// command is run from.
     #[arg(long)]
-    pub millstone_config_path: PathBuf,
+    pub adp_millstone_config_path: Option<PathBuf>,
```

Comment:
Added a separate config path because the hostnames are different for both containers so I can't use a shared file with the same `target` in the millstone yaml.